cvat-ui/src/components/annotation-page/annotation-filter/annotation-filter-item/annotation-filter-item.tsx

```diff
@@ -0,0 +1,115 @@
+// Copyright (C) 2020-2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import { Tag } from 'antd';
```

Comment:
Let's use the same import policy, now project uses modular import: ``import Tag from /antd/lib/Tag``
For information: https://nsisodiya.medium.com/reduce-size-in-ant-design-bundle-size-c35dad9ce3a8